src/nodes/Token.hack

```diff
@@ -144,7 +144,7 @@ abstract class Token extends Node {
     $trailing_list = __Private\fold_map(
       /* HH_IGNORE_ERROR[4110] */ $json['trailing'],
       ($j, $p) ==> Trivia::fromJSON($j, $file, $p, $source, 'Node'),
-      ($j, $p) ==> $j as shape('width' => int, ...) + $p,
+      ($j, $p) ==> $j as KeyedContainer<_, _>['width'] as int + $p,
```

Comment:
ergh. would be really good for us to make dicts pass shape typechecks until we get a typed json_decode

Reply:
I would be open for that. This only works because we are only indexing once. If we needed more from this structure, it would get pretty noisy pretty fast.